connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/IdentityReplicationIntegrationTest.java

```diff
@@ -108,16 +108,16 @@ public void testReplication() throws Exception {
         Map<TopicPartition, OffsetAndMetadata> backupOffsets = waitForCheckpointOnAllPartitions(
                 backupClient, consumerGroupName, PRIMARY_CLUSTER_ALIAS, "test-topic-1");
 
-        // Failover consumer group to backup cluster.
+        // Fail-over consumer group to back up cluster.
```

Comment:
Nit: this should actually be "fail over" (two words) to signify the verb, instead of "failover" (one word), which is the noun.

For the record, I'd be fine with leaving this as-is, but if we're going to change it, we should change it to the right thing.